### ISSUE TITLE
f9 to card list and turn card. 

### ISSUE THREAD

**Issue by jolugama** [octgn/OCTGN#1918]
when I turn a card with f9, card list appears. f9 is used to rotate card and appear card list.
Card list must have another shortcut

**Comment by kellyelton** [comment]
Hotkeys are specific to each game. I would recommend opening a ticket in that games github repository, or get in contact with the developer if that doesn't exist.

**Comment by jolugama** [comment]
Yes, but list card is new. I think that if it hits some hot key, and future games, I think it shouldn't have a hotkey. That is, the f9 of lotr may crash, but with the f9 of list card prevents creating future hotkeys in future games. I think that the list card hotkey should be removed, which I think is not so important, since we have it in the menu above and it is being given too much importance.

**Comment by jolugama** [comment]
Summary: I do not believe that whatever is added new, has to have a hotkey, since it damages old games and limits new ones. If it is in a menu, it gives enough importance. It is not that it indicates that it damages lotr, it is that it damages each new section to future games.

**Comment by kellyelton** [comment]
I appreciate your concern. Considering this hokey has been in use for awhile now, I'm not sure it makes sense to change it at this point, unless enough people get behind it and are interested in it changing.

I would also say that by your logic ("I think that the list card hotkey should be removed, which I think is not so important, since we have it in the menu above and it is being given too much importance."), it doesn't make sense to have hotkeys for games, since you can right click on cards/groups.

**Comment by kellyelton** [comment]
If more people come forward with the same concern we can look into it, I'm just hesitant to give up a hotkey.
